application/templates/job.yaml

```
@@ -37,13 +37,7 @@ spec:
       annotations: {{ toYaml . | nindent 8 }}
       {{- end }}
     spec:
-      {{- if $.Values.rbac.enabled }}
-      {{- if $.Values.rbac.serviceAccount.name }}
-      serviceAccountName: {{ $.Values.rbac.serviceAccount.name }}
-        {{- else }}
-      serviceAccountName: {{ template "application.name" $ }}
-      {{- end }}
-      {{- end }}
+      serviceAccountName: {{ include "application.serviceAccountName" $ }}
```

Same comment as in the other threads.

```suggestion
      automountServiceAccount: {{ $.Values.rbac.enabled }}
      serviceAccountName: {{ include "application.serviceAccountName" $ }}
```